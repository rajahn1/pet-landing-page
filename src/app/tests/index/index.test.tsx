import { render, screen } from "@testing-library/react";
import Home from "@/app/page";

describe("Home", () => {
  it("renders a heading", () => {
    const { container } = render(<Home />);

    const heading = screen.getByRole("heading", {
      name: /Hello World/i,
    });

    expect(heading).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
