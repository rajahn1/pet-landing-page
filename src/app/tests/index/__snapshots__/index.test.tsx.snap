// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
<div>
  <div
    class="w-screen flex flex-row"
  >
    <h1>
       Hello World
    </h1>
    <div>
      Hello World 2
    </div>
    <div>
      Hello World 3
    </div>
  </div>
</div>
`;
